tests/is/L42/tests/TestToNameUrl.java

```diff
@@ -51,6 +51,6 @@ void checkErr(String in) {
   @Test public void test42Url1(){check("AdamsTowel",
       "NameUrl[fullName=localhost[###]AdamsTowel.L42, fullPath=[###]/localhost/AdamsTowel.L42]");}
   @Test public void test42Url2(){check("L42.is/AdamsTowel",
-      "NameUrl[fullName=https://github.com/Language42/is/blob/HEAD/AdamsTowel.L42?raw=true, fullPath=https://github.com/Language42/is/blob/HEAD/AdamsTowel.L42?raw=true]");}
+      "NameUrl[fullName=https://github.com/"+ToNameUrl.l42IsRepoPath+"/blob/HEAD/AdamsTowel.L42?raw=true, fullPath=https://github.com/"+ToNameUrl.l42IsRepoPath+"/blob/HEAD/AdamsTowel.L42?raw=true]");}

```

Comment:
How is a field working here? I think you where making a more complex setting with a class+the reflection...